pom.xml

```diff
@@ -161,14 +161,19 @@
             <artifactId>junit</artifactId>
             <version>4.13.2</version>
             <scope>test</scope>
+        </dependency>
+         <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
+            <version>3.12.4</version>
```

Comment:
I'd extract this as a property (`mockito.version`), given that it's the same version for both `core` and `inline`.